nbsite/nbbuild.py

```diff
@@ -274,7 +277,7 @@ def nb_to_html(nb_path, preprocessors=[]):
 
 # TODO: (does it matter) have lost NotebookRunner.MIME_MAP['application/vnd.bokehjs_load.v0+json'] = 'html'
 # TODO: (does it matter) have lost NotebookRunner.MIME_MAP['application/vnd.bokehjs_exec.v0+json'] = 'html'
-def evaluate_notebook(nb_path, dest_path=None, skip_exceptions=False,substring=None, end=None, skip_execute=None,skip_output=None, offset=0, timeout=300, ipython_startup=None,patterns_to_take_with_me=None):
+def evaluate_notebook(nb_path, dest_path=None, skip_exceptions=False,substring=None, end=None, skip_execute=None,skip_output=None, offset=0, no_interactivity_warning=False, timeout=300, ipython_startup=None,patterns_to_take_with_me=None):
```

Comment:
I find options starting with `no_` to be very confusing; is this a warning about "no interactivity", or a directive not to warn about interactivity being limited?  Seems entirely ambiguous. I'd rather see an `interactivity_warning`.  It should probably default to True, which seems like the safest setting given that nbsite is explicitly about generating static HTML pages from dynamic notebooks, and thus it's always possible that there's dynamic content that won't work unless we explicitly declare that there isn't.

Reply:
It annoys me too, but there is a reason. The typical way to enable a boolean sphinx directive is to simply mention it e.g one file I was using to test used:

```rst
.. notebook:: my-org ../examples/Development.ipynb
    :offset: 0
    :no_interactivity_warning:
```

This means the expected *default* is False (the presence of the option indicates true). You can see a number of such boolean directives [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html) and none of them take an explicit argument. From what I can tell, you can't force a value (I tried `:no_interactivity_warning: False` for instance and the parser choked) and haven't found any such examples online.

Of course I can make it such that the presence of `:interactivity_warning:` *disables* the warning but then option seems misnamed and confusing, hence `:no_interactivity_warning:`. Alternatively, I could make it take an int argument so that you can disable with `:interactivity_warning: 0` (or negative?) and `:interactivity_warning: 1` (or positive?) to enable - but that seems like an abuse of integer arguments when really it should be a bool. Hope that makes sense!

Reply:
Ok, how about `disable_interactivity_warning`? I think that's unambiguous.